x-pack/filebeat/module/zeek/signature/test/signature-json.log-expected.json

```diff
@@ -0,0 +1,32 @@
+[
+    {
+        "@timestamp": "2019-11-06T09:03:00.989Z",
+        "destination.address": "192.168.116.172",
+        "destination.ip": "192.168.116.172",
+        "destination.port": 445,
+        "event.dataset": "zeek.intel",
```

Comment:
This doesn't appear to be the properly generated file. To generate this with the expected pipeline run, you can run:

```
make mage
```
followed by

```
GENERATE=true PYTEST_ADDOPTS="-k zeek" TESTING_FILEBEAT_FILESETS=signature mage -v pythonIntegTest
```

do note that because it runs the integration tests in a docker environment, the test run will likely take quite some time.

Reply:
I had issues with trying to run this but maybe because I was trying to run it in a container.

Reply:
@legoguy1000 so this looks like it still needs to be regenerated. Would you like me to regenerate and just push to your branch?

Reply:
I'm going to try to run it today.  If i still have issues, I'll post here.

Reply:
I'm getting this error.  Running it in a Ubuntu WSL 2 system on windows 10.
```
alex@DESKTOP-8FT9NF9:/mnt/c/Users/adr82/Documents/GitHub/beats$ GENERATE=true PYTEST_ADDOPTS="-k zeek" TESTING_FILEBEAT_FILESETS=signature mage -v pythonIntegTest
Unknown target specified: "pythonIntegTest"
```

Reply:
Also ran into these issues
```
alex@DESKTOP-8FT9NF9:/mnt/c/Users/adr82/Documents/GitHub/beats/filebeat$ make create-fields MODULE=zeek FILESET=signature

mage generate:fields
Error: cannot read pipeline: open /mnt/c/Users/adr82/Documents/GitHub/beats/filebeat/module/zeek/signature/ingest/pipeline.json: no such file or directory
make: *** [Makefile:28: create-fields] Error 1
alex@DESKTOP-8FT9NF9:/mnt/c/Users/adr82/Documents/GitHub/beats/filebeat$ 
alex@DESKTOP-8FT9NF9:/mnt/c/Users/adr82/Documents/GitHub/beats/filebeat$ cd ../x-pack/filebeat/
alex@DESKTOP-8FT9NF9:/mnt/c/Users/adr82/Documents/GitHub/beats/x-pack/filebeat$ make create-fields MODULE=zeek FILESET=signature
make: *** No rule to make target 'create-fields'.  Stop.
```

Reply:
@andrewkroh any ideas?

Reply:
The command @andrewstucki provided should be run from within `beats/x-pack/filebeat`. That will update the generated test files. Running `mage update` from there will update some generated files after you edit the `fields.yml` files.

Reply:
ok looks like i got it to work.

Reply:
I reran the `make update` and pushed again.  I think that should solve the error from one of the CI jobs.